smarts/env/rllib_hiway_env.py

```diff
@@ -37,8 +37,6 @@ class RLlibHiWayEnv(MultiAgentEnv):
                 used to specify a specific sumo port (default None)
             timestep_sec:
                 the step length for all components of the simulation (default 0.1)
-            visdom:
```

Comment:
Why removed this comment?

Reply:
We weren't actually supporting this parameter at this point.